troubleshooting/known_issues/Gigabye_Missing_Redfish_Data.md

```diff
@@ -0,0 +1,24 @@
+### Gigabyte BMC Missing Redfish Data
+
+Follow this procedure if you notice data from Gigabyte nodes are missing from Hardware State Manager(HSM) or other CSM tools.
+Make sure the firmware and BIOS of the Gigabyte node is at the latest supported level.

```

Comment:
This might be useful to put in a prereq section at the end of the intro. For example:

#### Prerequisite

* Make sure the firmware and BIOS of the Gigabyte node are at the latest supported level.

###### Reset the Gigabyte BMC

Reply:
Updated